recipes/pyflashtext/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pyflashtext" %}
+{% set version = "2.7.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: a5c19b11f4d17c524e66a273635b7858b581b23ce7478030965d6a3facf4be50
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
```

Comment:
Change to `script: "{{ PYTHON }} -m pip install . -vv"` as shown in the `example/meta.yaml`.